### DISCUSSION TITLE
Question: Customizable Shell TabBar Tab icon (Android + iOS)

### DISCUSSION THREAD

**Discussion by lnaie** [dotnet/maui#22404] [Q&A]
### Description

Hi,

I was wondering how is this achievable with MAUI?
![1666927](https://user-images.githubusercontent.com/1276077/175348782-820e037b-9c0e-4a0a-b01c-561ba10a0df8.png)
I think it's from an old version of the YouTube app.

It looks like the text is ignored, the image has almost double the size of the other tabs, and it's been vertically centered.

### Public API Changes

Don't have.

### Intended Use-Case

Well, to build a prettier tab bar. I was wondering why is not there yet. :)

**Comment by simphonydeveloper** [comment]
have an answer?

**Comment by VladislavAntonyuk** [comment]
https://vladislavantonyuk.github.io/articles/Adding-custom-action-button-to-.NET-MAUI-Shell-TabBar/
https://vladislavantonyuk.github.io/articles/Customizing-.NET-MAUI-Shell/
